drivers/interrupt_controller/Kconfig.eclic

```
@@ -0,0 +1,18 @@
+# Nuclei ECLIC interrupt-controller configuration
+
+# Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_NUCLEI_ECLIC = nuclei,eclic
+
+config HAS_NUCLEI_ECLIC
+	bool
+	help
+	  Indicate that the platform has ECLIC.
+
+config NUCLEI_ECLIC
+	bool "Extended Core Interrupt Controller (eclic)"
```

I couldn't find out what `L` in the middle stand for, so I looked it up.

Do you mean:
```suggestion
	bool "Enhanced Core Local Interrupt Controller (ECLIC)"
```

Ref: https://doc.nucleisys.com/nuclei_spec/isa/eclic.html

---

It is my mistakes. Thank you!